projects/valtimo/components/src/lib/components/topbar/topbar.component.scss

```diff
@@ -36,9 +36,9 @@
 
```

Comment:
@rutger-h 
A preview of the logo with 8px padding is now on https://v10.test.valtimo.nl/

Reply:
Ok, check. Top and bottom are ok now. Left should be in line with the menu, 10px
![Screenshot 2023-01-09 at 09 26 25](https://user-images.githubusercontent.com/59374585/211266332-04c821af-20da-4581-aa39-be2e9f92c515.png)

Reply:
I've hange the padding-left to 16 px to make it inline with the menu. A preview is now deployed